Local development problem

ran npm install and now running npm start gives this message

[13:56:59] Requiring external module babel-register
Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
Warning: Accessing createClass via the main React package is deprecated, and will be removed in React v16.0. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class v15.* is available on npm as a temporary, drop-in replacement. For more info see https://fb.me/react-create-class
/Users/devionvillegas/Documents/GitHub/one-click-hugo-cms/node_modules/core-js/library/fn/json/stringify.js:4
  return $JSON.stringify.apply($JSON, arguments);
                         ^

TypeError: Converting circular structure to JSON
    at Object.stringify (<anonymous>)
    at stringify (/Users/devionvillegas/Documents/GitHub/one-click-hugo-cms/node_modules/core-js/library/fn/json/stringify.js:4:26)
    at compile (/Users/devionvillegas/Documents/GitHub/one-click-hugo-cms/node_modules/babel-register/lib/node.js:105:42)
    at loader (/Users/devionvillegas/Documents/GitHub/one-click-hugo-cms/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/devionvillegas/Documents/GitHub/one-click-hugo-cms/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! victor-hugo@1.0.0 start: `gulp server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the victor-hugo@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
     "babel-plugin-transform-object-assign": "^6.8.0",
     "babel-plugin-transform-object-rest-spread": "^6.8.0",
     "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.11.1",
     "babel-register": "^6.11.6",
     "browser-sync": "^2.13.0",
     "css-loader": "^0.23.1",

